src/app/integrations/detail-page/detail.component.ts

```diff
@@ -402,4 +408,12 @@ export class IntegrationsDetailComponent extends IntegrationViewBase
     this.integrationSubscription.unsubscribe();
     this.routeSubscription.unsubscribe();
   }
+
+  exportIntegration() {
+    const id = this.i.id;
+    const url = this.config.getSettings().apiEndpoint + '/integrations/' + id + '/export.zip';
```

Comment:
It'd be better to use restangular to build this URL really, for example:

https://github.com/syndesisio/syndesis-ui/blob/master/src/app/store/integration-support.service.ts#L33

as building URLs this way can get broken really easily...